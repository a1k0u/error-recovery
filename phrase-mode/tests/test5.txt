OUT 1 + a + 10

a = 1

b = 100

OUT a + b

c = 10; k = 10; OUT c + k
