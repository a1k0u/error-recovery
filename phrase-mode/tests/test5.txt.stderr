warning: expected ';' at 1 line.
warning: expected ';' at 3 line.
warning: expected ';' at 5 line.
warning: expected ';' at 7 line.
warning: expected ';' at 9 line.
